src/components/NucleotideEntropy.tsx

```diff
@@ -0,0 +1,706 @@
+    let startIndex = getBrushIndex(geneRange, filteredEntropy, plotData.sequenceType).startIndex;
+    let stopIndex = getBrushIndex(geneRange, filteredEntropy, plotData.sequenceType).stopIndex;
+    return (
+      <>
+        <ResponsiveContainer width='100%' height='100%'>
+          <BarChart
+            key={(startIndex || 0) + (stopIndex || 0)} //This fixes a weird bug where the plot doesn't redraw when the brush indexes are changed
+            width={500}
+            height={500}
+            data={filteredEntropy}
+            barCategoryGap={0}
+            barGap={0}
+            margin={{
+              top: 30,
+              right: 20,
+              left: 20,
+              bottom: 5,
+            }}
+          >
+            <XAxis dataKey='position' tickFormatter={formatXAxis} />
+            <YAxis ticks={[0, 0.5, 1]} />
+            <Tooltip
+              content={<CustomTooltip />}
+              allowEscapeViewBox={{ y: true }}
+              wrapperStyle={toolTipStyle}
+            />
+            <Legend />
+            <Bar shape={CustomBar} dataKey='entropy' legendType='none'></Bar>
+            <Brush
+              dataKey='name'
+              height={10}
+              stroke={colors.active}
+              travellerWidth={10}
+              gap={10}
+              startIndex={startIndex}
+              endIndex={stopIndex}
+            />
+          </BarChart>
+        </ResponsiveContainer>
+      </>
+    );
+  } else {
+    return (
+      <>
+        <ResponsiveContainer width='100%' height='100%'>
+          <LineChart
+            width={500}
+            height={500}
+            data={plotData.timeData}
+            margin={{
+              top: 30,
+              right: 20,
+              left: 20,
+              bottom: 5,
+            }}
+          >
+            <XAxis
+            dataKey='day'
+            scale='time'
+            type='number'
+            tickFormatter={formatDate}
+            domain={[(dataMin: any) => dataMin, () => plotData.timeData[plotData.timeData.length - 1]?.day]}
+            ticks={plotData.ticks}
+            xAxisId='day'
+          />
+            <YAxis  />
+            <Tooltip
+            formatter={(value: string) => Number(value).toFixed(6)}
+            labelFormatter={label => {
+            return 'Week starting on: ' + formatDate(label);
+            }}
+        />
+            <Legend />
+            {selectedGenes.map((gene: GeneOption) => (
+              <Line
+                xAxisId='day'
+                type='monotone'
+                dataKey={gene.value}
+                strokeWidth={3}
+                dot={false}
+                stroke={gene.color}
+                isAnimationActive={false}
+                key={gene.value}
+                legendType='none'
+              />
+            ))}
+          </LineChart>
+        </ResponsiveContainer>
+      </>
+    );
+  }
+};
+
+const CalculateEntropy = (
```

Comment:
This should have a name starting with lower case. Only React components should start with a capital letter.